pagopa/pn-frontend-e2e
# Pn 9156 e 2 e destinatario pg recapiti cortesia modifica email pg

## Short description

Created new test: PG changes the email

## List of changes proposed in this pull request

- created new test: PG changes the courtesy email
- created new file with methods for both PG and PF for recapiti section
- eliminated yaml logic when adding courtesy mail in backgroundTest

## How to test

Simply run `mvn clean test -D"cucumber.filter.tags=@TA_modificaEmailPG"` and verify if the test works fine.

- [ x ] Does this PR require a manual test in AWS CodeBuild?
